.mailmap

```diff
@@ -18,7 +18,9 @@ Dan Wendlandt <dan@isovalent.com>
 Daniel Qian <qsj.daniel@gmail.com>
 Darshan Chaudhary <deathbullet@gmail.com>
 Devarshi Sathiya <devarshisathiya5@gmail.com>
+El-Fadel Bonfoh <elfadel@accuknox.com>
```

Comment:
I got this wrong in my suggested entry, it should instead be
```suggestion
El-Fadel Bonfoh <elfadel@accuknox.com> <bonfohelfadel@gmail.com>
```
as per my comment below.